src/dnf-plugins/subscription-manager.py

```diff
@@ -142,7 +142,17 @@ def _warnOrGiveUsageMessage(self):
             if msg:
                 logger.info(msg)
 
+    def sack(self):
+        # If command demands resolving, it should run transaction. This cannot
+        # be run in a Plugin.config hook as currently the Plugin.config is
+        # called before Command.configure.
+        if not self.cli.demands.resolving:
+            self._package_profile()
```

Comment:
Should profile action be activated by the same config attribute ('package_profile_on_trans') as transaction one?